### PR TITLE
Updated the Lambda functions so we can sync data with CQC in seperate requests

### PR DESCRIPTION
**Pull request by aaron-russell**

### CODE REVIEW

File: CQCLocationAPICall/aws/secrets.js [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    } else {
+      return myLocalSecrets.DB_ROOT_CRT;
+    }
+  } else {
+    throw new Error('Unknown secrets');
+  }
+};
+
+
+module.exports.initialiseSecrets = initialiseSecrets;
```
Comment (pjshumphreys) [inline CQCLocationAPICall/aws/secrets.js]: for this you could do: 

module.exports = {
initialiseSecrets,
dbHost,
dbPass,
jwtSecret,
slackUrl,
govNotify,
dbAppUserKey,
dbAppUserCertificate,
dbAppRootCertificate,
adminUrl
};

File: CQCLocationAPICall/handler.js [hunk 2]
```diff
@@ -44,7 +45,7 @@ module.exports.handler =  async (event, context) => {
           });
           break;
       }
-
+      models.sequelize.close();
```
Comment (pjshumphreys) [inline CQCLocationAPICall/handler.js]: I think the code looks better with this. Presumably this is to close off any connections this model open still right?

File: CQCLocationAPICall/handler.js [hunk 3]
```diff
@@ -76,7 +77,7 @@ module.exports.handler =  async (event, context) => {
           changes.push(response.data.changes[i]);
         }
       } while (nextPage != null);
-
+      console.log('There are ' + changes.length + ' changes');
```
Comment (pjshumphreys) [inline CQCLocationAPICall/handler.js]: Nice. It's a shame we don't have consistent logging across the app so we can do more than this

File: CQCLocationAPICall/handler.js [hunk 4]
```diff
@@ -125,20 +129,20 @@ module.exports.handler =  async (event, context) => {
   //
   async function populateAllLocations(url){
     let response;
-    let nextPage='/locations?page=22&perPage=1000';
-  
+    let nextPage='/locations?page=1&perPage=1000';
```
Comment (pjshumphreys) [inline CQCLocationAPICall/handler.js]: interesting, Why was it using page 22 before?

File: CQCLocationAPICall/models/index.js [hunk 5]
```diff
@@ -1,20 +1,60 @@
-'use strict';
```
Comment (pjshumphreys) [inline CQCLocationAPICall/models/index.js]: I'd prefer if this line was kept

File: CQCLocationAPICall/serverless.yml [hunk 6]
```diff
@@ -19,11 +19,11 @@ service: cqc-location-call # NOTE: update this with your service name
 
 provider:
   name: aws
-  runtime: nodejs8.10
+  runtime: nodejs10.x
```
Comment (pjshumphreys) [inline CQCLocationAPICall/serverless.yml]: cool cool

File: CQCLocationAPICall/handler.js (hunk 2 above)
Comment (aaron-russell) [reply CQCLocationAPICall/handler.js]: Yeah, obviously we don't want this for the API but we need to close the connections before the process is finished.

File: CQCLocationAPICall/handler.js (hunk 4 above)
Comment (aaron-russell) [reply CQCLocationAPICall/handler.js]: The comment above describes it, it's a total hack, this part of the function is if you want o completely refresh your data from scrath. So if the process fails you are able to containue from where it failed

File: CQCLocationAPICall/models/index.js (hunk 5 above)
Comment (aaron-russell) [reply CQCLocationAPICall/models/index.js]: Okay, I'll put this back in

File: CQCLocationAPICall/aws/secrets.js (hunk 1 above)
Comment (aaron-russell) [reply CQCLocationAPICall/aws/secrets.js]: This was just lifted steaight off the API to get everything working.

**Pull request merged**